validation/policies/io/konveyor/forklift/openstack/host_devices.rego

```diff
@@ -12,6 +12,6 @@ concerns[flag] {
 	flag := {
 		"category": "Warning",
 		"label": "VM has mapped host devices",
-		"assessment": "The VM is configured with hardware devices mapped from the host. This functionality is not currently supported by OpenShift Virtualization.",
+		"assessment": "The VM is configured with hardware devices mapped from the host. This functionality is not currently supported by OpenShift Virtualization. The VM can be migrated it will not have any host devices attached to it in the target environment.",
```

Comment:
```suggestion
		"assessment": "The VM is configured with hardware devices mapped from the host. This functionality is not currently supported by OpenShift Virtualization. The VM can be migrated but it will not have any host device attached to it in the target environment.",
```

Reply:
Done!